### PR TITLE
Valero Conversion II

### PR DESCRIPTION
**Pull request by h-mayorquin**
Adding some other aspects as the author provides information for us.

### CODE REVIEW

File: buzsaki_lab_to_nwb/valero/behaviorinterface.py [hunk 1]
```diff
@@ -151,7 +151,28 @@ def run_conversion(self, nwbfile: NWBFile, metadata: dict, stub_test: bool = Fal
...
+            "QWake_ThDt": "Quiet wakefulness esimated with higher theta/delta ratio",
+            "QWake_noRipples_ThDt": "Quite wakefulness without ripples, estimated with higher theta/delta ratio",
+            "NREM_ThDt": "Non-REM sleep, estimated with higher theta/delta ratio",
+            "NREM_noRipples_ThDt": "Non-REM sleep without ripples, estimated with higher theta/delta ratio",
+        }
+        description = (
+            "Sleep state of the subject."
+            "Estimated using `https://github.com/buzsakilab/buzcode/tree/master/detectors/detectStates/SleepScoreMaster`"
+        )
+        import json
```
Comment (CodyCBakerPhD) [inline buzsaki_lab_to_nwb/valero/behaviorinterface.py:171]: Can you move this import to global level?

Comment (h-mayorquin) [reply buzsaki_lab_to_nwb/valero/behaviorinterface.py]: Yep, done.

File: buzsaki_lab_to_nwb/valero/eventsinterface.py [hunk 2]
```diff
@@ -48,7 +48,7 @@ def run_conversion(self, nwbfile: NWBFile, metadata: dict, stub_test: bool = Fal
 
         # Create TimeIntervals
         name = "UpDownStatesTimeIntervals"
-        description = "TBD"
+        description = "TBD"  # TODO Cody, what do you think is a good description of this?
```
Comment (h-mayorquin) [inline buzsaki_lab_to_nwb/valero/eventsinterface.py:51]: Looking for your opinion here @CodyCBakerPhD .

**Review by h-mayorquin — COMMENTED** [review]
This is ready. It adds the latest information that the author gave in the email and modified the video interface to use the latest functions for synch.

I will change the functions from `run_conversion` to `add_to_nwb` in another PR.

File: buzsaki_lab_to_nwb/valero/eventsinterface.py (hunk 2 above)
Comment (CodyCBakerPhD) [reply buzsaki_lab_to_nwb/valero/eventsinterface.py:51]: lol

Y'know, I really have no clue. It's a very specific scientific classification used by this lab (not sure if I've seen it used anywhere else...)

I normally do the 'prettier placeholder' for this; `"Classifications of UP/DOWN neural states."`

As for the actual description of what those are, what they mean, how they are classified - I'd recommend asking the author or deferring to the publication

**Comment by CodyCBakerPhD** [comment]
This looks really good! Do you think we'd be able to get a draft upload (with or without raw) within the week?

File: buzsaki_lab_to_nwb/valero/eventsinterface.py (hunk 2 above)
Comment (CodyCBakerPhD) [reply buzsaki_lab_to_nwb/valero/eventsinterface.py:51]: Fine to add in follow-up if you want

**Comment by h-mayorquin** [comment]
I don't have access to all the sessions

> This looks really good! Do you think we'd be able to get a draft upload (with or without raw) within the week?

If we have access to all the data and I managed to download it, that should be doable.

**Pull request merged**